policy/delegations.md

```diff
@@ -0,0 +1,222 @@
+
+Delegations will be made for a set time period. This period is known as an epoch. The epoch will be marked by a Juno Network block number which will be provided after applications are scored and delegations are made. The epoch length is determined by the SubDAO.
+
+## 1.7 Determination of individual delegation amounts
+
+The total amount that will be delegated to each validator will be a function of the total amount allocated to be delegated by the Juno Development Fund, the total score earned by the validator (resultant from assessment of application), and the points multiplier to be applied as per section 1.8.
+
+## 1.8 Points multiplier
+
+To help promote a healthy distribution of stake across the validator set, a multiplier will be applied to the validator's total points.
+
+The multiplier will range from 1.0 to 1.1 where the lowest-ranked validator will have a 1.1x multiplier applied and the 11th-ranked validator will have a 1x multiplier.
+
+| Rank  | Multiplier    |
+| ---   | ---           |
+| 11    | 1             |
+| rank  | $$1+((rank-10)* 0.000714285714286)$$ |
+| 150   | 1.1           |
+
+
+
+# 2. Delegation Criteria
+
+The delegation criteria have been developed with the best interest of the Juno Network and the Juno Community in mind. The criteria aim to incentivize desirable outcomes for the Juno Network including:
+
+- Decentralized, reliable and trusted network of validators.
+- Retaining full network history.
+- Robust IBC relayer network.
+- Robust and accessible network of RPC and API nodes.
+- Development of the core network binary.
+- Development of ecosystem dApps and tooling.
+- Participation in network governance and governance discussions.
+- Social participation in the network ecosystem.
+
+## 2.1 Minimum Requirements
+
+A Validator must satisfy the minimum requirements of the program. These requirements are scored as ***pass/fail***. If any of the minimum requirements are not met, the validator will not be considered for a delegation and the application will be rejected.
+
+If a Validator has received a delegation and is found to no longer meet the minimum requirements at any point during the epoch, the delegation will be removed.
+
+***2.1.1 Submission of application*** – An application must be submitted to the SubDAO as detailed in section 1.5
+
+***2.1.2 Validator information*** – The validator must have accurate on-chain information including:
+
+***2.1.2 (a)*** a minimum of one (1) form of contact information (e.g Twitter, Telegram, Email etc).
+
+***2.1.2 (b)*** a security contact. A verification email will be sent to the address provided for confirmation.
+
+***2.1.2 (c)*** a logo/avatar has been submitted to the Mintscan explorer GitHub repository and merged.
+
+***2.1.3 Maximum commission*** – The validator must have a commission rate lower than or equal to 10%.
+
+***2.1.4 Maximum rank*** – The validator must be ranked outside the top 10 validators, ranked by bonded voting power.
```

Comment:
***2.1.4 Maximum rank*** – The validator must be ranked outside the top 10 validators, ranked by bonded voting power.

Should we add "at the time of application submission/review."

Primarily for the ones ranked 8-12 who are most likely to move in/out of the minimum criteria.